### DISCUSSION TITLE
240v split phase, plus 3rd out of phase leg?

### DISCUSSION THREAD

**Discussion by dalklein** [emporia-vue-local/esphome#63] [Q&A] [answered]
I have just set up the Vue2 in an off-grid panel with most of the house loads, set up per the typical US 240V split phase instructions.  It's working great (well, except the clumsy cloud & app), and I plan to convert it to ESPhome.   (Thanks flaviu et al !) 

Next to it is a on-grid panel with a couple of loads not fully defected.  The grid and off-grid systems don't stay in phase.  Monitoring a fixed resistive load from the grid panel looks like this, reading the correct 3800w, when the two panels happen to be beating in phase.
![image](https://user-images.githubusercontent.com/49771139/201825425-f67fb581-09ad-4d0e-abbc-2c12ff2753a8.png)

If I hook up the blue L3 voltage sense line to the grid panel L1,  then I should be able to measure loads there too.  It doesn't matter that L3 voltage and grid panel loads will be out of phase, and constantly changing phase, relative to the L1,L2 loads in the off-grid panel.  Just do the configuration appropriately.  Right?    Thanks!!!

**Comment by flaviut** [comment, marked as answer]
Right, that makes sense! But a bit of explanation of how this works:

1. there's a microchip that does the readings and sends back all the data
2. ESPHome just applies some calibration & sends those readings to where you want them.

Unfortunately, the other chip can only handle tracking a single frequency. The various phases can be unchanging multiples of that main frequency, but cannot drift. I'm not sure why this was done like this, but unfortunately fixing this is a really big software project.

Also, I'm not an electrician, but is having the vue connected to two different "grids" compliant with the electrical code?

**Comment by dalklein** [comment]
Yes the phases will be drifting, beating in/out of sync, although typically both really close to 60hz.      The power is line voltage for that phase * current for a given load, so maybe it'll work, I think I'll try.  It does use the voltage of the particular phase?  But maybe not, like you say about the measurement chip.

That's a good question to check:  Are the voltage sense leads only high impedance, but they do power the Vue also so they might not be completely innocuously low impedance.  I could turn off the grid panel, with the Vue wired up, and make sure it doesn't induce any voltage at all in the grid panel, from the off-grid panel through the Vue voltage sense connections.  One does need to be careful to not connect any circuit to both grid & off-grid at same time, it's effectively a short when they're out of phase and the off-grid equipment will lose vs. the grid, depending how fast circuit breakers trip.  The tap for the voltage sense to the grid panel I'd have a small 2 or 3A fuse on it.

**Reply by clowrey** [reply]
I would purchase another emporia Vue for each panel, they are incredibly affordable for what they are. I have flashed over 8 of them for ESPhome now.
